demo/client/backend/config.json

```diff
@@ -0,0 +1,13 @@
+{
+    "connection_profile_filename":"../gateway/localhost/connection_profile.json",
+    "channel_name":"mychannel",
+    "smart_contract_name":"mockcc",
+    "wallet": "../gateway/localhost/wallet",
+    "admin_user_name":"org1admin",
+
+    "users" : [
+    {"userName":  "A-Telecom",   "userRole" : "auction.bidder"} ,
```

Comment:
> probably makes sense to add the MSPid and just set it right now to Org1 everywhere: That way it would mark it somewhat explicitly already now as a place we might have to eventually adapt when moving to to multi-org. That said, registeredUsers.sh as consumers will anyway have to change for multi-peer and maybe to move node-client from utils/docker-compose/node-sdk, so not sure whether it might better to hold off to the time we move to multi-org overall and we have to the "bigger picture" on what multi-org really means? So my feel is that for now might just leave it at most by adding mspid here but not (yet!) try to adapt the scripts correspondingly ...

I will set MSPID to "Org1MSP".  This is the value set in fabric-private-chaincode/utils/docker-compose/network-config/docker-compose.yml.